server/src/entity/Restaurant.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  OneToMany,
+  JoinTable,
+} from 'typeorm';
+import { Menu } from './Menu';
+
+export enum cuisineTypes {
```

Comment:
You should uncomment all of the cuisineTypes

Reply:
NO